### PR TITLE
Statusbox blog article

### PR DESCRIPTION
**Pull request by mfuhrmann**
I'm not sure, if the GIF file works in Grav. Github can't display it. Or I made a mistake.

### CODE REVIEW

File: user/pages/05.blog/2017-08-14-statusbox/item.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+“As this feature was intended to be a small enhancement but a lot of issues were raised while developing this issue, I am now going to burry it.”
+... and I was really sad to hear that. Certainly I could understand it.
+
+But good things come to those who wait! The OpenNMS guys revived the Statusbox feature at the DevJam in Montreal and they followed through to complete it!
+But to cut a long story short...
+
+![donut](donut.gif)
+
+In my opinion this new feature is one of the most handy features since several updates - from the point of an OpenNMS user who works everyday with ONMS. It improves the usability and brings a new fancy look at the starting page.
```
Comment (mvrueden) [inline user/pages/05.blog/2017-08-14-statusbox/item.md]: Do we follow the one sentence per line rule here as well? If so it is not followed (-:

In addition I wouldn't use ONMS but OpenNMS instead.

File: user/pages/05.blog/2017-08-14-statusbox/item.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+But that's not all! This donut graphs scale perfectly! It doesn't matter if you have a small or big environment. You will get a perfect overview of the actual state.
+
+There is a [demo instance](https://demo.opennms.eu) (demo/demo) for presentation purposes. Login and check it out!
```
Comment (mvrueden) [inline user/pages/05.blog/2017-08-14-statusbox/item.md]: (username: `demo`, password: `demo`) or (login: `demo/demo`).

File: user/pages/05.blog/2017-08-14-statusbox/item.md (hunk 1 above)
Comment (mfuhrmann) [reply user/pages/05.blog/2017-08-14-statusbox/item.md]: You are right! I will fix that.

File: user/pages/05.blog/2017-08-14-statusbox/item.md (hunk 2 above)
Comment (mfuhrmann) [reply user/pages/05.blog/2017-08-14-statusbox/item.md]: Yes. It could look more beautyful.

**Review by mvrueden — APPROVED** [review]

**Pull request merged**